client/src/test-runner/runner.ts

```diff
@@ -557,3 +551,33 @@ function createDecorations(
     }
   });
 }
+
+function resolveElmBinaries(
+  configured: IElmBinaries,
+  ...roots: vscode.Uri[]
+): IElmBinaries {
+  const rootPaths = Array.from(new Set(roots.map((r) => r.fsPath)).values());
+  return <IElmBinaries>{
+    elmTest:
+      configured.elmTest ??
+      rootPaths
+        .map((r) => findLocalNpmBinary("elm-test", r))
+        .filter((p) => p)[0],
+    elmMake:
```

Comment:
I think we can drop this, as it's 0.18 behavior and we specifically said, this extension is for 0.19 only.